spec/models/like_spec.rb

```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly include unit test for like model

Reply:
Thank you ,its done .